include/tvm/relay/expr.h

```diff
@@ -166,6 +166,18 @@ class VarNode : public ExprNode {
 
 RELAY_DEFINE_NODE_REF(Var, VarNode, Expr);
 
+struct VarHash {
```

Comment:
You can simply use NodeHash and NodeEqual without introducing these

Reply:
NodeHash and NodeEqual is wrong, pointer equality for var is bad as var is based by id. In fact I only realize after using NodeHash and getting error.

Reply:
OK, please add more comment on they check equivalence by id.